packages/linter/package.json

```diff
@@ -24,10 +24,12 @@
   "homepage": "https://nx.dev",
   "builders": "./builders.json",
   "peerDependencies": {
-    "@nrwl/workspace": "*"
+    "@nrwl/workspace": "*",
+    "@angular-devkit/build-angular": ">=0.800.0",
```

Comment:
This package cannot depend on `build-angular` because we don't want to bring angular into non-angular workspaces.

Reply:
I actually don't think this is needed at all - thank you, I removed it 👍